bin/deploy-vhost

```diff
@@ -135,16 +135,30 @@ sub get_pgpw_string {
 
 sub one_db_conf_php {
     my ($params, $database, $internal) = @_;
-    my $port_line = '';
+    my $port_line       = '';
+    my $db_ro_host_line = '';
+    my $port_ro_line    = '';
     if ($params->{type} eq 'psql') {
         $port_line = "define('OPTION_$params->{prefix}_DB_PORT', $params->{port});";
+        if ($params->{replica}) {
+            $db_ro_host_line = "define('OPTION_$params->{prefix}_DB_RO_HOST', '$params->{replica}.ukcod.org.uk');";
+            # Hot standby usually on 5433, unless it's been failed over at some point.
+            if ($params->{port} eq '5432') {
+                $port_ro_line = "define('OPTION_$params->{prefix}_DB_RO_PORT', 5433);";
```

Comment:
To avoid the repetition of most of this line, set `$ro_port = $params->{port} == 5432 ? 5433 : 5432;` first and then use that? Happens again below as well.